draft-ietf-quic-transport.md

```diff
@@ -1069,6 +1069,19 @@ to cease using the connection IDs when requested can result in connection
 failures, as the issuing endpoint might be unable to continue using the
 connection IDs with the active connection.
 
+An endpoint SHOULD elect to limit the number of outstanding RETIRE_CONNECTION_ID
+frames to bound the necessary state. In order to allow a peer to retire all
+previously issued connection IDs, the limit on the number of outstanding
+RETIRE_CONNECTION_IDs SHOULD be at least the active_connection_id_limit.  If the
+number of outstanding RETIRE_CONNECTION_IDs exceeds twice the
+active_connection_id_limit, the connection MAY be closed with a connection error
+of type CONNECTION_ID_LIMIT_ERROR.
```

Comment:
```suggestion
RETIRE_CONNECTION_IDs SHOULD be at least the active_connection_id_limit.
```
That sentence is unnecessary.  If a peer starts forcing extra retirement, then this is no different than any of the multitude of other pathological behaviours they might exhibit.  It doesn't need special text.

Reply:
Well @ianswett just added the ERROR bit, so I'd like this input on this.

Reply:
Assuming we move forward with this PR, this is what I intend our implementation to do.  I realize that 'A server's gotta do what a server's gotta do' but I'd prefer an explicit expectation for ease of debugging.

Reply:
In that case, I would prefer something a little more terse and less prescriptive.

> An endpoint MAY choose to treat having too many connection IDs in need of retirement as a connection error of type CONNECTION_ID_LIMIT_ERROR.